src/Templates/CSharpStandardTemplatesSetup/source.extension.vsixmanifest

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="45474e92-a5d6-4ffb-8689-d7e8064cec90" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>CSharp Standard Templates</DisplayName>
+    <Description xml:space="preserve">CSharp Standard Templates</Description>
```

Comment:
Normally yes (in the extension manager dialog). However since these VSIXes will be marked as Productcomponent they won't show up in the package manager dialog. So no.